Lib/test/support/__init__.py

```diff
@@ -58,7 +58,8 @@
     "LOOPBACK_TIMEOUT", "INTERNET_TIMEOUT", "SHORT_TIMEOUT", "LONG_TIMEOUT",
     "Py_DEBUG", "EXCEEDS_RECURSION_LIMIT", "Py_C_RECURSION_LIMIT",
     "skip_on_s390x",
-    "without_optimizer",
+    "without_optimizer", "set_not_colorized_globally", "reset_colorized_globally",
```

Comment:
Doesn't exist and not used anywhere? It fails `test_all` 

```suggestion
    "without_optimizer", "reset_colorized_globally",
```